src/System.Private.ServiceModel/tests/Scenarios/Binding/Http/BasicHttpBindingTests.cs

```diff
@@ -4,17 +4,20 @@
 using System;
 using System.ServiceModel;
 using System.ServiceModel.Channels;
+using System.ServiceModel.Tests.Common;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 using TestTypes;
 using Xunit;
 
-public static class Binding_Http_BasicHttpBindingTests
+public class Binding_Http_BasicHttpBindingTests : OuterLoopTests
```

Comment:
We don't have a test initialization story yet.

For the Bridge initialization I was thinking of something along the lines of a statically available instance of some singleton bridge client, like:
```c#
public class BridgeClient {
    public static BridgeClient Instance {get{...}}
    private void ConfigureIfNeeded() {...}
    public string GetOrCreateService(string serviceName) {
       ConfigureIfNeeded();
       ... use HttpClient to interact with Bridge ...
}

public static class Endpoints {
    public static string HttpBaseAddress_Basic{
       return BridgeClient.Instance.GetOrCreateService("wcfservice") + "/Basic";
    }
}
```
Then the existing tests would just work without modification. The BridgeClient could just Assert that the bridge is running so when it isn't tests fail with a simple message like "Unable to contact the test bridge at URL xyz".